internal/pkg/warewulfd/daemon.go

```diff
@@ -86,7 +85,7 @@ func DaemonStatus() error {
 
 func DaemonReload() error {
 	if !util.IsFile(WAREWULFD_PIDFILE) {
-		wwlog.Printf(wwlog.INFO, "Warewulf daemon process not running (%s)\n", WAREWULFD_PIDFILE)
+		fmt.Printf("Warewulf daemon process not running (%s)\n", WAREWULFD_PIDFILE)
 		return nil
```

Comment:
Again, should return the error.

```suggestion
		return errors.New(fmt.Sprintf("warewulf daemon process is not runnin (%s)", WAREWULFD_PIDFILE))
```